monkey/agent_plugins/credentials_collectors/chrome/src/linux_credentials_database_selector.py

```diff
@@ -1,11 +1,116 @@
-from typing import Collection
+import logging
+import os
+import pwd
+from pathlib import Path
+from typing import Collection, Dict, Sequence, Set
 
 from .utils import BrowserCredentialsDatabasePath
 
+logger = logging.getLogger(__name__)
+
+LinuxUsername = str
+UserDirectories = Dict[LinuxUsername, Path]
+
+DEFAULT_MASTER_KEY = "peanuts".encode()
+
+LOGIN_DATABASE_FILENAME = "Login Data"
+CHROMIUM_BASED_DB_PATHS = [
+    ("Google Chrome", ".config/google-chrome"),
+    ("Chromium", ".config/chromium"),
+    ("Brave", ".config/BraveSoftware/Brave-Browser"),
+    ("SlimJet", ".config/slimjet"),
+    ("Dissenter Browser", ".config/GabAI/Dissenter-Browser"),
+    ("Vivaldi", ".config/vivaldi"),
+    ("Microsoft Edge (Dev)", ".config/microsoft-edge-dev"),
+    ("Microsoft Edge (Beta)", ".config/microsoft-edge-beta"),
+    ("Microsoft Edge", ".config/microsoft-edge"),
+]
 
-class LinuxCredentialsDatabaseSelector:
-    def __init__(self):
-        pass
 
+class LinuxCredentialsDatabaseSelector:
     def __call__(self) -> Collection[BrowserCredentialsDatabasePath]:
-        return []
+        database_paths: Set[BrowserCredentialsDatabasePath] = set()
+        for profile_dir in self._get_browser_database_paths():
```

Comment:
The paths being returned from `self._get_browser_database_paths()` are not profile directory paths. They're browser directory paths for each user on the system. The name `profile_dir` makes it seem like you're talking about the profiles in within the browser.